definitions/npm/formik_v0.11.x/flow_v0.53.x-/formik_v0.11.x.js

```diff
@@ -9,33 +9,58 @@ declare module "formik" {
     [field: $Keys<Values>]: boolean
   };
 
-  declare export type FormikActions<Values> = {
+  declare export interface FormikActions<Values> {
```

Comment:
What motivated the switch to an interface? I'm not even seeing `FormikActions` being used?

Reply:
Like the Typescript definition, there are some overloaded method signatures in FormikActions. As far as I know, flow supports that for interfaces, but doesn't for object types (https://github.com/facebook/flow/issues/1556#issuecomment-200051475)

Reply:
Makes sense; can you document that justification? Otherwise there might be some churn.

Reply:
Sure, added a comment explaining the use of interface

Reply:
I prefer type instead of interfaces

Reply:
I would use type if they supported overloaded method signatures. As it is, interface seems like the right choice.